components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/SSOConsentServiceImpl.java

```diff
@@ -544,7 +546,7 @@ private PIICategory addPIICategoryForClaim(ClaimMetaData claim) throws SSOConsen
             piiCategory = getConsentManager().addPIICategory(piiCategoryInput);
         } catch (ConsentManagementException e) {
             throw new SSOConsentServiceException("Consent PII category error", "Error while adding" +
-                                                                      " PII category:" + DEFAULT_PURPOSE_CATEGORY, e);
+                    " PII category:" + DEFAULT_PURPOSE_CATEGORY, e);
```

Comment:
" PII category:" can be moved to the previous line. 
To improve the readability would be better to add a space after ":"

Reply:
Fixed in 06271985d0d22192422630521503f04db25e5eff.